### ISSUE TITLE
Mail view: show labels

### ISSUE THREAD

**Issue by elie222** [elie222/inbox-zero#149]
Show labels on threads. Similar to Gmail.

Note: there are rules for when labels show. Each label has this set.
![CleanShot 2024-03-03 at 09 49 37@2x](https://github.com/elie222/inbox-zero/assets/3090527/e3dd0313-3aae-4021-9232-34814f8a2b85)

**Comment by Deeptanshu-sankhwar** [comment]
@elie222 how can I navigate to the thread section with the display for labels in the app?

**Comment by elie222** [comment]
I don't understand the question. Could you rephrase, please

**Comment by Deeptanshu-sankhwar** [comment]
Sure @elie222 , this screenshot which you've attached in this issue, is it a snippet from inbox-zero? If yes, then where can I find it, so I can make the required changes to it?

**Comment by varuntree** [comment]
Do you mean, we need to add the options to each of the labels below to set rules? 
![image](https://github.com/elie222/inbox-zero/assets/117532982/a2ca58a2-b043-4394-9247-c12c6c9cc15e)

Something like this?
![image](https://github.com/elie222/inbox-zero/assets/117532982/48c96834-e80a-4c19-9b86-04e1dd99ed58)

**Comment by Inayat27** [comment]
Hello @elie222 can i work on issue #149

**Comment by harshalvk** [comment]
Working on issue, [Mail view: show labels #149]
Hello @elie222 sir, **is this how you would like the labels to be displayed? Please let me know if any changes are needed.**
![image](https://github.com/user-attachments/assets/ac343f9e-6496-4d3e-b675-2f2b00b19e7c)

**Comment by elie222** [comment]
Done. Thanks!
